roles/webserver/templates/nginx_unicorn.j2

```diff
@@ -18,7 +18,7 @@ server {
 
   include {{ app }}_ssl;
 
-  server_name www.{{ domain }};
+  server_name {{sub_domain}} {{ domain }};
```

Comment:
Yes. I thought I had reverted this change. I had seen the server_name `www.127.0.0.1` and thought that would be a problem, but subsequently realized it's only a name - and the second `server_name` ended up being a duplicate of the first (which broke provisioning - or at least broke something along the way) when done this way. I think this is just a line of code I forgot to revert back.